docs: update docker-compose (#25332)
apache/superset: docs/docs/installation/installing-superset-using-docker-compose.mdx

```bash
docker compose up
```
:::tip

> When running in development mode the `superset-node` container needs to finish building assets in order for the UI to render properly. If you would just like to try out Superset without making any code changes follow the steps documented for `production` or a specific version bellow.

Reviewer: ```suggestion
When running in development mode the `superset-node` container needs to finish building assets in order for the UI to render properly. If you would just like to try out Superset without making any code changes follow the steps documented for `production` or a specific version below.
```

Alternatively, you can also run a specific version of Superset by first checking out
the branch/tag, and then starting `docker-compose` with the `TAG` variable.
For example, to run the 3.0.0 version, run the following commands:

```bash
git checkout 3.0.0
TAG=3.0.0 docker-compose -f docker-compose-non-dev.yml pull

> TAG=0.0.0 docker-compose -f docker-compose-non-dev.yml up

Reviewer: ```suggestion
TAG=3.0.0 docker-compose -f docker-compose-non-dev.yml up
```

> For example, to run the 2.1.0 version, run the following commands:

Reviewer 2: + 1
```suggestion
For example, to run the 3.0.0 version, run the following commands:
```
